packages/vitest/src/integrations/coverage/base.ts

```diff
@@ -0,0 +1,22 @@
+import type { ExistingRawSourceMap, TransformPluginContext } from 'rollup'
```

Comment:
Importing these rollup types seems to cause out-of-memory locally, CI is OK. These errors started showing up after merging the `browser` feature. 

```
v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool)
```

I've run into similar issues in other projects before. I think rollup is not that memory efficient when it comes to bundling multiple entries.

Reply:
Build seems to work with `export NODE_OPTIONS="--max-old-space-size=8192"`.